scene/resources/particles_material.cpp

```diff
@@ -1513,6 +1747,14 @@ void ParticlesMaterial::_bind_methods() {
 	BIND_ENUM_CONSTANT(EMISSION_SHAPE_RING);
 	BIND_ENUM_CONSTANT(EMISSION_SHAPE_MAX);
 
+	BIND_ENUM_CONSTANT(PARAM_TURB_CONTRAST);
+	BIND_ENUM_CONSTANT(PARAM_TURB_SCALE);
+	BIND_ENUM_CONSTANT(PARAM_TURB_SPEED_RAND);
+	BIND_ENUM_CONSTANT(PARAM_TURB_POS_INFLUENCE);
+	BIND_ENUM_CONSTANT(PARAM_TURB_VEL_INFLUENCE);
+	BIND_ENUM_CONSTANT(PARAM_TURB_INFLUENCE);
```

Comment:
This should be bound together with the other `Parameter` enums, otherwise it's out of sync in the docs.